Library/Services/INfieldSurveyLanguageTranslationsService.cs

```diff
@@ -0,0 +1,77 @@
+﻿//    This file is part of Nfield.SDK.
+//
+//    Nfield.SDK is free software: you can redistribute it and/or modify
+//    it under the terms of the GNU Lesser General Public License as published by
+//    the Free Software Foundation, either version 3 of the License, or
+//    (at your option) any later version.
+//
+//    Nfield.SDK is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//    GNU Lesser General Public License for more details.
+//
+//    You should have received a copy of the GNU Lesser General Public License
+//    along with Nfield.SDK.  If not, see <http://www.gnu.org/licenses/>.
+
+using System.Linq;
+using System.Threading.Tasks;
+using Nfield.Models;
+using Nfield.SDK.Models;
+
+namespace Nfield.Services
+{
+    /// <summary>
+    /// Represents a set of methods to read and update survey languages.
+    /// </summary>
+    public interface INfieldSurveyLanguageTranslationsService
+    {
+        #region CRUD on Language Translations
+
+        /// <summary>
+        /// Gets all languages and their translations for a survey, as a queryable object.
+        /// </summary>
+        /// <param name="surveyId">The survey for which language names and ids are returned</param>
+        /// <exception cref="T:System.AggregateException"></exception>
+        /// The aggregate exception can contain:
+        /// <exception cref="Nfield.Exceptions.NfieldErrorException"></exception>
+        /// <exception cref="Nfield.Exceptions.NfieldHttpResponseException"></exception>
+        Task<IQueryable<SurveyLanguageTranslations>> QueryAsync(string surveyId);
+
+        /// <summary>
+        /// Adds a new language and all of its translations.
+        /// </summary>
+        /// <param name="surveyId">the survey to add this language to</param>
+        /// <param name="languageId">A new id for the language to add</param>
```

Comment:
The add language shouldn't have a languageId

Reply:
It's part of the model?

I thought we agreed in our API plans that IDs would be taken out of the models as they are usually used as parameter in the URL REST path